code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1387,8 +1388,13 @@
 			var/obj/item/organ/heart/our_heart = M.getorganslot(ORGAN_SLOT_HEART)
 			our_heart.applyOrganDamage(1)
 
-// can be synthesized on station rather than bought. made by grinding a banana peel, heating it up, then mixing the banana peel powder with salglu
-/datum/reagent/medicine/coagulant/weak
-	name = "Synthi-Sanguirite"
-	description = "A synthetic coagulant used to help bleeding wounds clot faster. Not quite as effective as name brand Sanguirite, especially on patients with lots of cuts."
-	clot_coeff_per_wound = 0.8
+// i googled "natural coagulant" and a couple of results came up for banana peels, so after precisely 30 more seconds of research, i now dub grinding banana peels good for your blood
+/datum/reagent/medicine/coagulant/banana_peel
```

Comment:
why not just use the existing banana juice reagent for this? other juices already heal things like brute, burn, tox, and oxy damage, so the schtick of banana juice could be healing cut wounds (and also directly healing clowns, like it already does).

Reply:
Read the comment ath, it's right there. Just read it. It's there for a reason.

Reply:
that doesn't mean it isn't bloat-y